src/ConnectedMode/UI/ManageConnections/ManageConnectionsViewModel.cs

```diff
@@ -36,6 +36,15 @@ public async Task LoadConnectionsWithProgressAsync()
             await ProgressReporterViewModel.ExecuteTaskWithProgressAsync(validationParams);
         }
 
+        public async Task DeleteConnectionWithProgressAsync(ConnectionViewModel connectionViewModel)
```

Comment:
Why not `RemoveConnectionWithProgressAsync`? 🤔  (Same question for every place that we use `delete` instead of `remove`)

Reply:
My personal preference was Delete. I can rename it to remove for consistency reasons.